### PR TITLE
feat: add shutdown pass-through command

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

**Comment by gregmagolan** [comment]
just print_install & mobile-install to go after this. I'm going to leave license out for now since that one we should discuss how to handle

File: docs/aspect.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Aspect CLI is a better frontend for running bazel
 * [aspect modquery](aspect_modquery.md)	 - Queries the Bzlmod external dependency graph.
 * [aspect query](aspect_query.md)	 - Executes a dependency graph query.
 * [aspect run](aspect_run.md)	 - Builds the specified target and runs it with the given arguments.
+* [aspect shutdown](aspect_shutdown.md)	 - Stops the bazel server.
```
Comment (alexeagle) [inline docs/aspect.md:32]: nit: looks like you didn't `pre-commit install`

**Review by alexeagle — APPROVED** [review]

File: docs/aspect.md (hunk 1 above)
Comment (gregmagolan) [reply docs/aspect.md:32]: hmm. the pre-commit hooks are running. you see something not formatted?

Comment (gregmagolan) [reply docs/aspect.md:32]: oh.. the spacing here. odd

Comment (alexeagle) [reply docs/aspect.md:32]: oh nevermind, the right hand side of the hyphens on this list was aligned before, but that's go fmt which automates that, I shouldn't expect Prettier to do so

Comment (gregmagolan) [reply docs/aspect.md:32]:
```
go fmt...................................................................Passed
Test shell scripts with shellcheck...................(no files to check)Skipped
buildifier...............................................................Passed
buildifier-lint..........................................................Passed
prettier.................................................................Passed
Insert license in comments...............................................Passed
commitizen check.........................................................Passed
[shutdown 13e7dcc] feat: add shutdown pass-through command
 9 files changed, 162 insertions(+), 1 deletion(-)
 create mode 100644 cmd/aspect/shutdown/BUILD.bazel
 create mode 100644 cmd/aspect/shutdown/shutdown.go
 create mode 100755 docs/aspect_shutdown.md
 create mode 100644 pkg/aspect/shutdown/BUILD.bazel
 create mode 100644 pkg/aspect/shutdown/shutdown.go
```
🤷

**Pull request merged**